### DISCUSSION TITLE
drag generated img to img input

### DISCUSSION THREAD

**Discussion by PhiGo67** [lllyasviel/Fooocus#1777] [Q&A] [answered]
I use fooocus in Firefox, dragging generated pics on img input doesn´t work at all in latest version.
Can you confirm?

**Comment by usigg3** [comment]
Not for me either. Does it work with other browsers?

**Comment by mashb1t** [comment, marked as answer]
This is a known issue, see https://github.com/lllyasviel/Fooocus/issues/1425
Works flawless with Chrome though!
